tidb-configuration-file.md

```diff
@@ -947,6 +947,12 @@ TiDB 服务状态相关配置。
 + 该值作为系统变量 [`tidb_enable_ddl`](/system-variables.md#tidb_enable_ddl-从-v630-版本开始引入) 的初始值。
 + 在 v6.3.0 之前，该功能由配置项 `run-ddl` 进行设置。
 
+### `tidb_enable_stats_owner` <span class="version-mark">从 v8.4.0 版本开始引入</span>
+
++ 用于表示该 tidb-server 是否可以成为 stats owner。
```

Comment:
```suggestion
+ 用于表示该 tidb-server 是否可以运行[统计信息自动更新](/statistics.md#自动更新)任务。
```
The term "stats owner" (or "status owner") is not documented in the TiDB documentation.

Reply:
@hawkingrei PTAL